copter/source/docs/traditional-helicopter-swashplate-setup.rst

```diff
@@ -155,9 +155,45 @@ that produces zero thrust. In Copter 3.6 and ealier, the minimum collective pitc
 could command was done with the parameter ``H_COL_LAND_MIN``. 
 
 Set :ref:`H_SV_MAN <H_SV_MAN>` to 3 to have the swashplate move to the mid position.  Adjust :ref:`H_COL_MID <H_COL_MID>` until the blade
-pitch angle measures the desired collective blade pitch that corresponds to zero thrust.  
+pitch angle measures the desired collective blade pitch that corresponds to zero thrust.
 
+.. note:: If the minimum or maximum collective is measured in ACRO (see below for impact of ACRO on collective pitch curve vs stabilized modes), the value will be less than those since some servo movement is reserved for cyclic. Of course, any swashplate tilt due to pilot or autopilot input will interefere with the measurements also.
 
+ACRO versus STABILIZED Collective Curves
+========================================
+
+ArduPilot provides a programmable curve for stabilized modes (all except ACRO). This curve is set by the ``IM_STB_COL_x`` parameters as shown in the graph below. Only ACRO mode uses the collective extremes that were setup above.
```

Comment:
The collective curve is for stabilized mode only. It is not used for any other mode

Reply:
then it is not used in the other stabilized modes, which is every mode besides ACRO? I thought they would inherit from stabilized mode, otherwise, you could be running different curves depending on the history of how and when ACRO was entered..if this is the case, I think it is incorrect behavior....

Reply:
Yes all the other modes use the stabilize mode for pitch roll and yaw but the non manual throttle modes or auto collective modes do not use the collective control range of stabilize. They are given the full range that is given to acro.

Reply:
so the curve selection is determined by STABILIZE or ACRO mode entry, and it use the Input handler for non-auto-colllective modes? so DRIFT mode (the other piilot controlled collective mode)  curve is determined by whatever was last used ACRO or STABILIZE?

Reply:
It reverts to the full collective range from min to max. There is no special curve or handler